presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/MetastoreUtil.java

```diff
@@ -145,7 +146,8 @@
     private static final String NUM_ROWS = "numRows";
     private static final String RAW_DATA_SIZE = "rawDataSize";
     private static final String TOTAL_SIZE = "totalSize";
-    private static final Set<String> STATS_PROPERTIES = ImmutableSet.of(NUM_FILES, NUM_ROWS, RAW_DATA_SIZE, TOTAL_SIZE);
+    private static final String TRANSIENT_LAST_DDL_TIME = "transient_lastDdlTime";
```

Comment:
what is this new parameter ? can you share some info about it

Reply:
This parameter represents the latest modified time.
We should exclude it when using updateStatisticsParameters, which can create a correct transient_lastDdlTime rather than copying the old transient_lastDdlTime to hive. 
In a nutshell, I add this parameter to acquire a correct transient_lastDdlTime.

Reply:
So this parameter is not used anywhere else in the code ? So will Metastore set this value ? I am trying to understand when this key will be added to partition parameters

Reply:
Yes, Presto doesn't use the parameter. The parameter is added to the partition of presto when selected from the partition of Hive.
If Presto doesn't have this value, the partition of Hive will set the value at the present time automatically, which is what we want when concerning alter action.

Reply:
done. Please check if it is appropriate?